recipe/meta.yaml

```diff
@@ -13,9 +13,7 @@ build:
   number: 0
   features:
     - vc9  # [win and py27]
-    - vc10  # [win and py34]
-    - vc14  # [win and py35]
-    - vc14  # [win and py36]
+    - vc14  # [win and py>=35]
```

Comment:
MSVC 2017 (vc141) binaries are backwards and forwards compatible with 2015 (vc140).